### ISSUE TITLE
ModuleNotFoundError: No module named 'jupyter_server.services.contents.tests'

### ISSUE THREAD

**Issue by andrewfulton9** [jupyter-server/jupyter_server#394]
I am trying to reuse the `jupyter_server` `ContentsManager` tests for a custom `ContentsManager` I am working on. I am trying to import them [according to the documentation here](https://jupyter-server.readthedocs.io/en/latest/developers/contents.html?highlight=tests#testing)
However when I try to import this I get 
``` traceback
ModuleNotFoundError: No module named 'jupyter_server.services.contents.tests'
```

my environment:
os: ubuntu 18.04
package_manager: conda
channel: conda-forge
python: 3.9.1
jupyterlab: 3.0.5
jupyter_server: 1.2.2

I think the documentation was probably just updated switching out `notebook` for `jupyter_server` when the `ContentsManager` class was moved, but the tests were restructured so that they are no longer accessible from that location. I assume either the tests should be added back to the importable package, or the docs just need to be updated to reflect the changes.

**Comment by echarles** [comment]
Quick feedbacks:

1. The tests have moved to the tests folder https://github.com/jupyter-server/jupyter_server/tree/master/tests so the doc is not correct IMHO
2. The pypi publication does not contain the test folder https://pypi.org/project/jupyter-server/#files

**Comment by Zsailer** [comment]
Yes, these docs are (currently) out of date. The `tests` module was moved outside the package in #152.

However, it might be worth discussing whether the `tests` module should be added back into the package. Your use-case is pretty compelling—a custom contents manager where you want to reuse jupyter_server's unit tests.

Thoughts?

**Comment by andrewfulton9** [comment]
I've just copied the tests over to a local file and have just started modifying them to test my custom `ContentsManager`, so i'm not sure yet how many of them I will be able to use "out of the box". I think they will at least be useful for understanding the scope that my `ContentsManager` should cover and building my own tests to cover that scope. Whether that warrants making the tests importable again, I'm not sure. It may be enough to just update the docs to point a link to the current tests for people to use as a baseline for writing their own.

**Comment by kenyaachon** [comment]
I was a bit surprised when I noticed I could no longer import `jupyter_server.services.contents.tests`. I found it easier to import the existing jupyter_server tests, since I am using a custom jupyter_config.py for my jupyter_server, rather than trying to manage my jupyter_server tests while staying up to date with the latest tests that are use for jupyter_server
